src/main/webapp/app/exam/participate/summary/exercises/quiz-exam-summary/quiz-exam-summary.component.ts

```diff
@@ -116,10 +116,11 @@ export class QuizExamSummaryComponent implements OnInit {
             const submittedAnswer = this.submission.submittedAnswers.find((answer) => {
                 return answer && answer.quizQuestion ? answer.quizQuestion.id === quizQuestionId : false;
             });
-            if (submittedAnswer && submittedAnswer.scoreInPoints) {
+            if (submittedAnswer && submittedAnswer.scoreInPoints !== undefined) {
```

Comment:
hmm that's weird, normally `submittedAnswer.scoreInPoints` is more than enough since `null, undefined, 0` are all evaluated to false

Reply:
Maybe the change on line 123 is enough to fix the bug?

Reply:
the change on line 123 was after the change of line 119. 
This would be enough, yes. But the line 119 was falsy to begin with ;)

Reply:
> hmm that's weird, normally `submittedAnswer.scoreInPoints` is more than enough since `null, undefined, 0` are all evaluated to false

Only after this change to check for undefined, I had added the `return 0` statement.

Reply:
Yeah that's correct but leaving `if (submittedAnswer && submittedAnswer.scoreInPoints)` would ensure that `scoreInPoints` is both defined and a positibe number which is what we want for the rounding :D